lib/ui/window/pointer_data_packet.cc

```diff
@@ -20,4 +20,14 @@ void PointerDataPacket::SetPointerData(size_t i, const PointerData& data) {
   memcpy(&data_[i * sizeof(PointerData)], &data, sizeof(PointerData));
 }
 
+PointerData PointerDataPacket::GetPointerData(size_t i) {
+  PointerData result;
+  memcpy(&result, &data_[i * sizeof(PointerData)], sizeof(PointerData));
+  return result;
+}
+
+size_t PointerDataPacket::Size() {
```

Comment:
Perhaps `GetLength` or `GetCount` instead? Also, the missing `const`. `size` is usually meant for byte sizes (like in the case of `data_` here.

So something like `size_t PointerDataPacket::GetLength() const;` perhaps.

Reply:
I will do that